src/Common/Middleware/AddDefaultQueryParamsMiddleware.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace AzureOss\Storage\Common\Middleware;
+
+use GuzzleHttp\Psr7\Query;
+use Psr\Http\Message\RequestInterface;
+
+/**
+ * @internal
+ */
+class AddDefaultQueryParamsMiddleware
```

Comment:
Final since we do not want people outside the library to extend it since it will cause BCs